### PR TITLE
feat(RTE): allow tabbing to next or prev element

### PR DESCRIPTION
**Pull request by fulopdaniel**
Previously on TAB the RTE blurred and removed focus. However TAB should also focus the next element in the focus tree, so in this PR I am implementing that.

### CODE REVIEW

File: src/components/RichTextEditor/helpers/forceToFocusElement.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const forceToFocusNextElement = (event: React.KeyboardEvent, forwards: boolean) => {
+    if (!document.activeElement) {
+        return;
+    }
+    event.preventDefault();
+    const focusableElementSelectors =
+        'button, [href], input, select, textarea, [role="textbox"], [tabindex]:not([tabindex="-1"]';
```
Comment (ragi96) [inline src/components/RichTextEditor/helpers/forceToFocusElement.ts:9]:
```suggestion
        ''a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, [tabindex="0"], [contenteditable]';
```

What do you think to adjust them a bit more, so disabled stuff is not focusable

Comment (imoutaharik) [reply src/components/RichTextEditor/helpers/forceToFocusElement.ts:9]: On the focusTrap hook I'm working on, the list of tabbable elements is like this:
export const TABBABLE_ELEMENTS = [
    'input:not([disabled]):not([type=hidden])',
    'select:not([disabled])',
    'textarea:not([disabled])',
    'button:not([disabled])',
    'a[href]',
    'area[href]',
    'summary',
    'iframe',
    'object',
    'embed',
    'audio[controls]',
    'video[controls]',
    '[contenteditable]',
    '[tabindex]:not([tabindex="-1"]):not([disabled])',
].join(':not([hidden]):not([tabindex="-1"]),');

Comment (fulopdaniel) [reply src/components/RichTextEditor/helpers/forceToFocusElement.ts]: Thanks, great list. Implemented it with some tweaks.

**Review by imoutaharik — APPROVED** [review]

File: src/components/RichTextEditor/RichTextEditor.tsx [hunk 2]
```diff
@@ -83,8 +83,7 @@ export const RichTextEditor = ({
         },
         onKeyDown: (event) => {
             if (event.code === 'Tab') {
-                // Forcing a blur event because of accessibility
-                forceToBlurActiveElement();
+                forceToFocusNextElement(event, !event.shiftKey);
```
Comment (celine-s) [inline src/components/RichTextEditor/RichTextEditor.tsx:86]: Is it now still possible to leave the rte again? I now cannot tab outside of the RTE anymore.

File: src/components/RichTextEditor/helpers/forceToFocusElement.ts (hunk 1 above)
Comment (ragi96) [reply src/components/RichTextEditor/helpers/forceToFocusElement.ts]: @imoutaharik thanks for that improved list.

**Review by celine-s — APPROVED** [review]
nice

**Pull request merged**